src/mixins/java/org/spongepowered/common/mixin/api/mcp/world/IWorldReaderMixin_API.java

```diff
@@ -103,6 +105,11 @@ default boolean hasLiquid(final int x, final int y, final int z) {
         return this.shadow$hasWater(new BlockPos(x, y, z));
     }
 
+    @Override
+    default boolean containsAnyLiquids(AABB aabb) {
+
```

Comment:
These default implemented methods should throw exceptions for us to look at to implement.

Reply:
implemented this one
the others that are not in invalid now throw exceptions

IWorldMixin_LocationBasedDataHolder_API and ServerWorldMixin_InteractableVolume_API
are in invalid so they should get looked at anyways
I just extracted the not implemented methods from the mixins there were in previously.